pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix

```
@@ -1,18 +1,18 @@
-{ stdenv, buildPythonApplication, fetchFromGitHub, pyxdg, pytest }:
+{ stdenv, buildPythonApplication, fetchFromGitHub, pyxdg, pytest, pytest-mock }:
 
 buildPythonApplication rec {
   pname   = "pass-git-helper";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromGitHub {
     owner  = "languitar";
     repo   = "pass-git-helper";
     rev    = "v${version}";
-    sha256 = "18nvwlp0w4aqj268wly60rnjzqw2d8jl0hbs6bkwp3hpzzz5g6yd";
+    sha256 = "sha256-GdsFPpBdoEaOCmdKxw5xTuFOcGFH94w5q/lV891lCUs=";
   };
 
   propagatedBuildInputs = [ pyxdg ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytest, pytest-mock ];
```

```suggestion
  checkInputs = [ pytest pytest-mock ];
```
As you can tell from the CI, lists are not comma-separated in Nix.